contrib/opentimelineio_contrib/application_plugins/rv/example_otio_reader/otio_reader.py

```diff
@@ -300,6 +355,64 @@ def _create_movieproc(time_range, kind="blank"):
     return movieproc
 
 
+def _add_source_bounds(media_ref, src, context):
+    if not hasattr(media_ref, "available_image_bounds"):
+        return
+
+    global_scale = context.get('global_scale')
+    global_translate = context.get('global_translate')
+    if not global_scale or not global_translate:
+        return
+
+    bounds = media_ref.available_image_bounds
+    if not bounds:
+        return
```

Comment:
If you accept the first suggestion in the function, I think this would be covered by that check and could be removed.